### ISSUE TITLE
Gradle dependency freezes gradle/Android studio.

### ISSUE THREAD

**Issue by LawAbidingNinja** [roomorama/Caldroid#231]
Hi Thomas I didn't think it possible but it seems the adding Caldroid as a gradle dependency freezes Android Studio/gradle

Steps to recreate:

1. Create a project in Android Studio
2. Add compile 'com.roomorama:caldroid:1.1.8' to the project's build.gradle

I am also connecting via a proxy (may be causing an issue, however I can download other libraries without a problem). 

I did notice that maven reports the project has 0 usages (http://mvnrepository.com/artifact/com.roomorama/caldroid). I find this fairly surprising considering the number of forks Caldroid has on Github. 

(This is far more likely a bug in gradle. I think I'm just going mad from trying to fix it the whole day.)

**Comment by thomasdao** [comment]
I tried `compile 'com.roomorama:caldroid:1.1.8'` to a new project and existing project and it can compile ok. Probably you can update to latest Android Studio?

**Comment by LawAbidingNinja** [comment]
I do have the latest Android studio, it looks like my proxy might be the issue (sorry about that).
I ended up to building offline, downloading the .aar manually and using
 `compile files ('libs/caldroid-1.1.8.aar')`. This resulted with the .aar's internal AndroidManifest.xml conflicting with my own, which was rather strange.

Presently I've just included the source code/res files manually. This seems to be working fine.
